packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -29,26 +34,51 @@ export const styles = (theme) => ({
     ...theme.typography.body2,
     minHeight: 'auto',
   },
+  indicatorWrapper: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  indicator: {
+    marginLeft: theme.spacing(2)
+  }
 });
 
 const MenuItem = React.forwardRef(function MenuItem(props, ref) {
   const {
+    children: childrenProp,
```

Comment:
why rename this?

Reply:
Because an older implementation collided and the alias never got removed, I think. It should be removed.